cookbook/ui_customization/how_to_customize_menu.rst

```diff
@@ -0,0 +1,87 @@
+How to customize the Back Office menu
+=====================================
+
+The Akeneo back office uses the Oro Platform Application Menu.
+
+The NavigationBundle automatically processes a YAML configuration file which must be named ``navigation.yml`` and located in ``Resources/config`` directory of the registered bundle.
+
+The configuration is placed under ``oro_menu_config`` tree.
+
+Menu items creation
+*******************
+
+New entries are created under the ``items`` key. Each item must have a ``label`` and a ``route`` (or an ``uri``).
+
+.. code-block:: yaml
+
+    # /src/Acme/Bundle/AppBundle/navigation.yml
+    oro_menu_config:
+        items:
+            reference_data:
+                label: 'Reference Data'
+                uri: '#'
+            brand:
+                label: 'Brands'
+                route: 'app_brands_index'
+            manufacturer:
+                label: 'Manufacturers'
+                route: 'app_manufacturers_index'
+
+The example above defines three menu items:
+ - The ``reference_data`` item consists of a label and the URI ``#``. This means that the item will not react on mouse click,
+   but can be used as a placeholder for nested menus;
+ - Both ``brand`` and ``manufacturer`` items reference and existing route. So, when we will click one of theses items, we will
+   get the corresponding page.
+
+There is the list of all available options for an item:
+
+.. code-block:: yaml
+
+    items: #menu items
+        <key>: # menu item identifier. used as default value for name, route and label, if it not set in options
+            aclResourceId                     # ACL resource Id
+            translateDomain: <domain_name>    # translation domain
+                translateParameters:          # translation parameters
+            label: <label>                    # label text or translation string template
+            name:  <name>                     # name of menu item, used as default for route
+            uri: <uri_string>                 # uri string, if no route parameter set
+            route: <route_name>               # route name for uri generation, if not set and uri not set - loads from key
+                routeParameters:              # router parameters
+            attributes: <attr_list>           # <li> item attributes
+            linkAttributes: <attr_list>       # <a> anchor attributes
+            labelAttributes: <attr_list>      # <span> attributes for text items without link
+            childrenAttributes: <attr_list>   # <ul> item attributes for nested lists
+            showNonAuthorized: <boolean>      # show for non-authorized users
+            display: <boolean>                # disable showing of menu item
+            displayChildren: <boolean>        # disable showing of menu item children
+
+Organize the Navigation trees
+*****************************
+
+The next step is to compose a tree of the menu items. All trees are build under the ``tree`` key.
```

Comment:
_to compose_ means _faire partie de_, you should say _The next step is to put together a tree for these menu items_